weblate/templates/accounts/contact.html

```diff
@@ -11,8 +11,9 @@
 <div class="container">
 
 <div class="alert alert-info">
-    {% blocktrans %}You can contact maintainers of this server here.{% endblocktrans %}
-    {% blocktrans %}If you are looking for support for Weblate or want to file bug reports check its website at {{ weblate_link }}.{% endblocktrans %}
+    {% blocktrans %}<ul><li>For questions about specific translation project, contact its maintainers directly, not by this form.</li>{% endblocktrans %}
+    {% blocktrans %}<li>You can only contact maintainers of the whole server here.</li>{% endblocktrans %}
+    {% blocktrans %}<li>If you are looking for support for Weblate, or want to file a bug report, check its website at {{ weblate_link }}.</li></ul>{% endblocktrans %}
```

Comment:
Please do not use HTML inside translatable strings (unless necessary):

```suggestion
  <ul>
    <li>{% blocktrans %}For questions about specific translation project, contact its maintainers directly, not by this form.{% endblocktrans %}</li>
    <li>{% blocktrans %}You can only contact maintainers of the whole server here.{% endblocktrans %}</li>
    <li>{% blocktrans %}<li>If you are looking for support for Weblate, or want to file a bug report, check its website at {{ weblate_link }}.{% endblocktrans %}</li>
  </ul>
```